admin/urls.go

```diff
@@ -319,8 +319,11 @@ func (u *URLs) ReportExport(org, project, report, token string) string {
 	return exportURL
 }
 
-// ReportEdit returns the URL for editing a report in the frontend.
-func (u *URLs) ReportEdit(org, project, report string) string {
+// ReportEdit returns the URL for editing a report in the frontend or unsubscribe for non-rill recipients.
+func (u *URLs) ReportEdit(org, project, report, recipient, token string) string {
```

Comment:
Can we avoid passing the recipient here and do a lookup instead? It's not so good to have PII in the URL string itself